tests/cstor/gtest/test_zrepl_prot.cc

```diff
@@ -1987,7 +1993,8 @@ TEST(ZvolResizeTest, ResizeZvol) {
 	ASSERT_EQ(hdr_in.len, 0);
 
 	// get the zvol size after
-	str = execCmd("zfs", std::string("get -Hpo value volsize ") + zvolname);
+	str = execCmd("zfs", std::string("get -Hpo value volsize ") +
+			zvolname + "_rebuild_clone");
```

Comment:
Can you also add a verification of resize once volume become healthy?

Reply:
Done